README.md

```diff
@@ -10,6 +10,8 @@
 
 **Arcadia-Science/metagenomics** is a pipeline for profiling metagenomes obtained through either Illumina or Nanopore technologies.
 
+![](img/workflow-overview.jpg)
+
```

Comment:
ooh pretty yay

Reply:
sorry just looked at the figure -- i think `taxannotate` should be `tax annotate` right? or is there an entry point with no space in it?

Reply:
hmm the module is called taxannotate so I left it that way for clarity?

Reply:
oh that makes total sense! perfect!